### PR TITLE
holds onto async requests if node isnt started

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

**Comment by andrewxhill** [comment]
well.. 2 things happen in this branch.

first: 1st is it unblocks the photogrid even if node isn't started.

as a result, you may do asyncimage requests prior to the node being available, so

second: it waits to send any asyncimage request until node is 'started'

**Comment by andrewxhill** [comment]
biggest possible error that a user could get into now is that they make it all the way through a share before the node is started

File: App/Components/AsyncImage.js [hunk 1]
```diff
@@ -53,3 +67,16 @@ export default class AsyncImage extends React.Component {
     }
   }
 }
+
+const mapStateToProps = (state, ownProps) => {
```
Comment (asutula) [inline App/Components/AsyncImage.js]: You don't have to make `AsyncImage` a redux connected component like this. You can derive this state>props for the node state up in `TextilePhotos`, then you just pass the `nodeState` as a prop into `PhotoGrid` and them again as a prop into `AsyncImage`.

**Comment by andrewxhill** [comment]
Cool, lazy there. Done now.

**Comment by andrewxhill** [comment]
gtg

**Pull request merged**